languages/csharp/exercises/concept/README.md

```diff
@@ -6,16 +6,16 @@ The concept exercises are based on this [list of concepts][reference-shared].
 
 These are the concept exercises that have currently been implemented, as well as the concepts they teach and their prerequisite concepts:
 
-| exercise                                                            | concepts                                   | prerequisites                                                                     |
-| ------------------------------------------------------------------- | ------------------------------------------ | --------------------------------------------------------------------------------- |
-| [`basics`][concept-exercise-basics]                                 | `basics`                                   | -                                                                                 |
-| [`dates`][concept-exercise-datetimes]                               | `datetimes`                                | `numbers`<br/>`strings`<br/>`classes`                                             |
-| [`enums`][concept-exercise-enums]                                   | `enums`                                    | `strings`<br/>`conditionals`                                                      |
-| [`flag-enums`][concept-exercise-flag-enums]                         | `flag-enums`<br/>`bit-manipulation`        | `enums`<br/>`attributes`</br>`integers`                                           |
-| [`floating-point-numbers`][concept-exercise-floating-point-numbers] | `floating-point-numbers`<br/>`while-loops` | `numbers`<br/>`conditionals`                                                      |
-| [`numbers`][concept-exercise-numbers]                               | `numbers`<br/>`conditionals`               | `basics`                                                                          |
-| [`properties`][concept-exercise-properties]                         | `properties`                               | `numbers`<br/>`floating-point-numbers`<br/>`classes`<br/>`enums`<br/>`exceptions` |
-| [`strings`][concept-exercise-strings]                               | `strings`                                  | `basics`                                                                          |
+| exercise                                                            | concepts                                                            | prerequisites                                                       |
```

Comment:
Your changes are based on a slightly older version of the README. It might be best to copy the [latest version](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/README.md) into this file and re-apply the changes.

Reply:
Here is the commit I've done to resolve this: https://github.com/exercism/v3/pull/864/commits/0aaf582ea4babaca821efd6fbc9374ffddc227b5

Reply:
I'll have to edit this again since we reduced the prerequisites in the design.md to strings and conditionals. Another thing, we'll change it from conditionals-switch-basic to constant-switch?